fix: Update docker compose to the last version (#1646)
stacks-sbtc/sbtc: docker/mainnet/README.md

This requires the chain-state for Bitcoin and Stacks to be present,
respectively, at `/mnt/bitcoin` and `/mnt/stacks`.

### sBTC signer and Stacks Node versions

> Always run last version of the docker images.

Reviewer: ```suggestion
Always run the latest version of the docker images.
```

> Run always last version of the docker images.
> You can find the sBTC images [here](https://github.com/stacks-sbtc/sbtc/pkgs/container/sbtc) and the latest versions of Stacks [here](https://hub.docker.com/r/blockstack/stacks-blockchain/tags).

Reviewer: ```suggestion
Always run the last version of the docker images.
You can find the sBTC images [here](https://github.com/stacks-sbtc/sbtc/pkgs/container/sbtc) and the latest version of Stacks Core image [here](https://hub.docker.com/r/blockstack/stacks-blockchain/tags).
```

> You can find the sBTC images [here](https://github.com/stacks-sbtc/sbtc/pkgs/container/sbtc) and the latest version of Stacks Core Image [here](https://hub.docker.com/r/blockstack/stacks-blockchain/tags).

Reviewer: ```suggestion
You can find the sBTC images [here](https://github.com/stacks-sbtc/sbtc/pkgs/container/sbtc) and the Stacks Core images [here](https://hub.docker.com/r/blockstack/stacks-blockchain/tags).
```
